README.md

```diff
@@ -25,6 +25,31 @@ This is the official Python client for interacting with our powerful [API](https
 
 ---
 
+
+
+##Table Of Contents
+
+**[1. Installation](#installation)**
+**[2. Getting Started](#getting-started)**
+**[3. Interacting with Datasets](#interacting-with-datasets)**
+**[4. Interacting with Inputs](#interacting-with-inputs)**
```

Comment:
Need a new line in between. Right now all 4 are on the same line.

Reply:
![image](https://github.com/Clarifai/clarifai-python/assets/147609006/ae14c6a3-f76a-4494-8734-5209f5a51728)

They are on separate lines when previewed.

Reply:
That's odd. Both my preview in VS Code and Github shows that they are on the same line....
![Screenshot 2023-10-30 at 11 52 04 AM](https://github.com/Clarifai/clarifai-python/assets/48971969/8baa2ff4-c3af-4d7b-a8c2-e6012bfd2c11)
^ VS Code

![Screenshot 2023-10-30 at 11 52 54 AM](https://github.com/Clarifai/clarifai-python/assets/48971969/4c67e79b-2ac6-43e1-94f1-6c1a861275a7)


^ Github. [Try click this link to see the preview in Github](https://github.com/Clarifai/clarifai-python/blob/a06ca909a198b7adcbc1da5079e1c2c81680713f/README.md).

Reply:
OK, I will check that out.

Reply:
I have made the changes. Can you check?